contracts/eth/nearprover/hardhat.config.js

```diff
@@ -9,9 +9,10 @@ const INFURA_API_KEY = process.env.INFURA_API_KEY || '';
 task('upgrade-provers-bridge-address-to', 'Upgrades the provided prover to use the bridge at the provided address')
     .addParam('prover', 'Prover address')
     .addParam('newBridge', 'The address of the new bridge')
+    .addParam('ledgerKey', 'The ledger key path to sign transactions', undefined, undefined, true)
```

Comment:
It's actually not "Ledger key", but rather "Ledger key path".
Please change the parameter name accordingly

Reply:
Also, please add a separate `make` command that uses this option